File: 프로그래머스/0/181895. 배열 만들기 3/배열 만들기 3.cs
using System;
using System.Collections.Generic;
public class Solution {
    public int[] solution(int[] arr, int[,] intervals) {
        List<int> answer = new List<int>();
        for(int i = 0 ; i < intervals.GetLength(0);i++)
        {
            int start = intervals[i,0];
            int end = intervals[i,1];
            for(int j = start;j <= end;j++)
            {
                answer.Add(arr[j]);
            }
        }
        return answer.ToArray();
    }
}

File: 프로그래머스/0/181895. 배열 만들기 3/배열 만들기 3.cpp
#include <string>
#include <vector>

using namespace std;

vector<int> solution(vector<int> arr, vector<vector<int>> intervals) {
    vector<int> answer;
    for(int i = 0; i < intervals.size(); i++)
    {
        int a = intervals[i][0];
        int b = intervals[i][1];
        for(a; a <= b; a++)
            answer.push_back(arr[a]);
    }
    return answer;
}